### ISSUE TITLE
[BUG] Use state_dict instead of named_parameters for updating policy in subprocess?

### ISSUE THREAD

**Issue by taochenshh** [pytorch/rl#995]
Update policy weights do not consider variables that are registered as buffer? For example, when using batch normalization, there are variables like running_mean, but they won't show up on named_parameters(), so the update_policy_weights won't correctly update the policy in the subprocess when using the collectors?
